extras/minimal-printf/Makefile

```diff
@@ -1,5 +1,5 @@
 # Just for manual testing
 .PHONY: test
 test: duk_minimal_printf.c
-	gcc -fno-stack-protector -m32 -otest -Wall -Wextra -Os -fomit-frame-pointer duk_minimal_printf.c test.c
+	gcc -std=c99 -Wall -Wextra -fno-stack-protector -m32 -Os -fomit-frame-pointer -otest duk_minimal_printf.c test.c
```

Comment:
Is there a reason this one in particular is compiled with `-fno-stack-protector`?  It seems to be the only one.